src/fastlane/release_actions/fastlane/AndroidAppsFastfile

```diff
@@ -0,0 +1,178 @@
+RELEASE_TAG_PREFIX = 'release_v'
+default_platform(:android)
+
+platform :android do |options|
+  gradle_properties_path = File.expand_path("#{Dir.pwd()}/../../gradle.properties")
+  gradle_project_root = File.dirname(gradle_properties_path)
+  change_log_path = "#{gradle_project_root}/CHANGELOG.md"
+  version_regex = /(\d*)\.(\d*)\.(\d*)(?:-unstable\.)?(\d*)?/
+  pr_body = %Q(
+## Release PR review checklist
+- [ ] Verify version name in gradle.properties
+- [ ] Verify CHANGELOG.md
+)
+
+  desc "Fetch all tags to ensure accurate calculation of the next release number."
+  before_all do 
+    sh('git', 'fetch', '--tags')
+    sh('git', 'fetch')
+  end
+
+  lane :configure_git_options do |options|
+    sh('git', 'config', 'user.email', options[:git_user_email])
+    sh('git', 'config', 'user.name', options[:git_user_name])
+  end
+
+  # This lane is used to perform tasks necessary to create a PR for a release
+  # of an Android-based library. To execute from your project, add the following to your
+  # Fastfile:
+  # 
+  # ```
+  # import_from_git(
+  #     url: 'git@github.com:aws-amplify/amplify-ci-support.git',
+  #     branch: 'android/fastlane-actions',
+  #     path: './src/fastlane/release_actions/fastlane/AndroidAppsFastfile'
+  # )
+  # ```
+  # 
+  # To execute, run the following command from the directory where your Gemfile is located:
+  # `bundle exec fastlane android create_next_release_pr repo:myrepos/myandroidlib product_name:"My awesome library" docfiles_to_update:"README.md" dryrun:`
+  # Options:
+  #   repo: Github repo in the owner/repo_name format.
+  #   product_name: Friendly name of the library you're releasing. This will be used in PRs and release notes 
+  #   docfiles_to_update: any text files containing code blocks that show users how to add gradle dependencies on your libraries.
+  #                       A regex replacement will be applied that will replace anything matching `implementation 'package_name:module_name:current_version'`
+  #                       with `implementation 'package_name:module_name:new_version'`.
+  #   dryrun: useful for testing localy. It runs the different steps but does not commit changes or creates the PR. Be sure to 
+  #           pass this parameter with a colon at the end (without a value afterwards). See sample above.
+  lane :create_next_release_pr do |options|
+    # This will be used in the commit and PR title
+    product_name = options[:product_name]
+    maven_group_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name)
+    docfiles_to_update = options.has_key?(:docfiles_to_update) ? options[:docfiles_to_update].split(' ') : []
+    files_to_commit = [gradle_properties_path, change_log_path]
```

Comment:
For `amplify-android`, we also need to bump `core-kotlin/gradle.properties`

Reply:
Ah, that's right. Let me see what I can come up with for that.